lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -43,6 +43,7 @@
 module KubernetesDeploy
   class DeployTask
     include KubeclientBuilder
+    extend KubernetesDeploy::StatsD::MeasureMethods
```

Comment:
The only real changes in this file are the new instrumentations via `measure_method`. The rest is just moving code around to package it into the methods I want to measure.